cmd/minishift/cmd/start.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/docker/machine/libmachine/provision"
 	"github.com/golang/glog"
 	"github.com/minishift/minishift/cmd/minishift/cmd/addon"
+	"github.com/minishift/minishift/cmd/minishift/cmd/config"
```

Comment:
What's about we alias 'config' to make it more explicit. Something like 'startFlag' or 'flags' or 'settings'?